vmdb/app/models/miq_vim_broker_worker.rb

```diff
@@ -20,7 +20,7 @@ def self.has_required_role?
   end
 
   def self.emses_to_monitor
-    EmsVmware.where(:zone_id => MiqServer.my_server.zone_id).includes(:authentications).select(&:authentication_valid?)
+    EmsVmware.where(:zone_id => MiqServer.my_server.zone_id).includes(:authentications).select(&:has_credentials?)
```

Comment:
I think this should really be calling authentication_status_ok? so we don't monitor ems's that we recently detected were unreachable, we had a bad password, etc.